tests/time_integration/ab_constants_02.cc

```diff
@@ -0,0 +1,59 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#include <iostream>
+
+#include <exadg/time_integration/ab_constants.h>
+
+// Check if AB time integration constants in case of adaptive timestepping
```

Comment:
"if" is too much?

Reply:
[0ff6c74](https://github.com/exadg/exadg/pull/469/commits/0ff6c7473ea0718357e4456919e541b6eb516f7c)